api/preprints/serializers.py

```diff
@@ -150,17 +160,17 @@ def update(self, preprint, validated_data):
             preprint.node.preprint_article_doi = validated_data['article_doi']
             save_node = True
 
+        if 'license_type' in validated_data or 'license' in validated_data:
```

Comment:
Ah OK. It would be worth adding a comment by the EZID update below that it should be called after all the updates are finished so that the metadata are correct.